src/plugins/charts/public/services/colors/colors_palette.test.ts

```diff
@@ -91,15 +90,16 @@ describe('Color Palette', () => {
   });
 
   it('should return the seed color array when input length is 72', () => {
-    expect(createColorPalette(num2)[71]).toBe(seedColors[71]);
+    expect(createColorPalette(num2).length).toBe(72);
+    expect(createColorPalette(num2)[71]).toBe('#BFB540');
```

Comment:
let's avoid the hard-coded values, as those will be subject to change with theme changes to OUI.